src/gsi/setuprad.f90

```diff
@@ -1,4 +1,7 @@
 module rad_setup
+! GSI fails to run using newer Intel compilers (i.e, Intel 2022)
+! with optimization: https://github.com/NOAA-EMC/GSI/issues/447
+!DIR$ OPTIMIZE:0
```

Comment:
```suggestion
! TODO:  Temporary until the root cause is identified and the issue mentioned above is resolved.
!DIR$ OPTIMIZE:0 
```

Reply:
Thanks, @aerorahul . I was also thinking to add something to the top-level README.md / changelog or whatever is appropriate for this repo if this change is acceptable and works as intended.